pylot/flags.py

```diff
@@ -121,7 +121,8 @@
                   'Enable recording of bbox annotated detector images')
 flags.DEFINE_bool('log_traffic_light_detector_output', False,
                   'Enable recording of bbox annotated tl detector images')
-
+flags.DEFINE_bool('log_imu', False,
+                  'Enable recording of IMU measurements')
```

Comment:
Why did we introduce the `log_imu` flag? Is it being used outside of `data_gatherer.py`?

Reply:
I agree. Is it because you want to visualize IMU? If so, then can you please use the visualize_imu flag that we have?

Reply:
I used `log_imu` to do some analysis when running scenario tests. For example, I tested MPC vs PID on a swerve trajectory and needed to log the IMU measurements in order to generate graphs. 

Lateral jerk, MPC vs PID:
![lat_jerk](https://user-images.githubusercontent.com/55596368/73223210-39a08e00-411a-11ea-82a8-0d9c98c93bdf.png)